actions.yaml

```
@@ -25,3 +17,61 @@ status-detail:
       type: boolean
       default: false
       description: a boolean indicating whether a unit should recompute all statuses.
+
+pre-refresh-check:
+  description: Check if charm is ready to refresh
+  additionalProperties: false
+
+force-refresh-start:
+  description: |
+    Potential of data loss and downtime
+
+    Force refresh of first unit
+
+    Must run with at least one of the parameters `=false`
+  params:
+    check-compatibility:
+      type: boolean
+      default: true
+      description: |
+        Potential of data loss and downtime
+
+        If `false`, force refresh if new version of MongoDB and/or charm is not compatible with previous version
```

```suggestion
        If `false`, force refresh if new version of Mongos and/or charm is not compatible with previous version
```
name here should match workload_name used in https://canonical-charm-refresh.readthedocs-hosted.com/latest/add-to-charm/instantiate/